src/parser.rs

```diff
@@ -63,14 +101,29 @@ pub fn normalize_crontab(input: &str) -> String {
 /// syntax definitions.
 ///
 /// ```rust
-/// use cronexpr::parse_crontab;
+/// use cronexpr::parse_crontab_with;
+/// use cronexpr::ParseOptions;
 ///
-/// parse_crontab("* * * * * Asia/Shanghai").unwrap();
-/// parse_crontab("2 4 * * * Asia/Shanghai").unwrap();
-/// parse_crontab("2 4 * * 0-6 Asia/Shanghai").unwrap();
-/// parse_crontab("2 4 */3 * 0-6 Asia/Shanghai").unwrap();
+/// let mut options = ParseOptions::default();
+/// parse_crontab_with("* * * * * Asia/Shanghai", options).unwrap();
```

Comment:
So I've taken [a somewhat different approach](https://github.com/maxcountryman/underway/blob/beef5ca8a82783a72caeafa283e4ad25ab3f8171/examples/scheduled/src/main.rs#L34). Granted, I'm also working around limitations in both `cron` and `jiff`. That said, you can imagine how this type could have methods for parsing crontab expressions of both formats transparently or methods that infer the time zone.

Reply:
It seems you get the timezone from jiff's db with its name in string. This is the same manner as in cronexpr.